galite-tests/src/test/resources/org/kopi/galite/test/Menu-fr_FR.xml

```diff
@@ -2,16 +2,22 @@
 
 <modules>
     <module ident="1000" label="Client" help="Clients management" />
-        <module ident="1001" label="Client" help="Clients management" />
-        <module ident="1010" label="Test" help="Suppliers management" />
-            <module ident="1101" label="Test form1" help="Test form" />
-            <module ident="1110" label="Test form2" help="Test form" />
+    <module ident="1001" label="Client" help="Clients management" />
+         <module ident="1010" label="Test" help="Suppliers management" />
+    <module ident="1101" label="Test form1" help="Test form" />
+         <module ident="1110" label="Test form2" help="Test form" />
     <module ident="2000" label="Command" help="Commands management" />
-        <module ident="2001" label="Command" help="Commands management" />
+         <module ident="2001" label="Command" help="Commands management" />
     <module ident="3000" label="Product" help="Products management" />
-        <module ident="3001" label="Product" help="Products management" />
+         <module ident="3001" label="Product" help="Products management" />
     <module ident="4000" label="Bill" help="Bills management" />
-        <module ident="4001" label="Bill" help="Bills management" />
+         <module ident="4001" label="Bill" help="Bills management" />
     <module ident="5000" label="Bill product" help="Bill of products" />
-        <module ident="5001" label="Bill product" help="Bill of products" />
+         <module ident="5001" label="Bill product" help="Bill of products" />
+    <module ident="6000" label="Stock" help="Stock management" />
+         <module ident="6001" label="Stock" help="Stock management" />
+    <module ident="7000" label="TaxRule" help="Tax Rules management" />
+         <module ident="7001" label="TaxRule" help="Tax Rules management" />
+    <module ident="8000" label="Provider" help="Providers management" />
+         <module ident="8001" label="Provider" help="Providers management" />
 </modules>
```

Comment:
1 - we need to check the hierarchy: because here under every menu, we have one form that has the same name of the menu!

2- if for example we have to class StoreAplication & TestAplication : can we specify hierarchy of menus for each application alone ?

3- can you find a way to run StoreAplication also on swing version 

4- can you make your form read and insert data into database